src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import '../../App.scss';
+
+export const Person = ({ person }) => {
+  const partnerType = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age !== null && person.age !== undefined && (
```

Comment:
```suggestion
      {person.age && (
```
This check can be simplified. It will return true for all positive values